.gitmodules

```diff
@@ -308,3 +308,6 @@
 	url = https://github.com/google/perfetto.git
 	branch = master
 	platforms = linux,android
+[submodule "third_party/json/repo"]
```

Comment:
we also have `third_party/jsoncpp` . What would be a path to unify and having only one library?

I.e. anything that jsoncpp cannot do but json can? And if so is it feasable to modify json to use jsoncpp? or could this code use jsoncpp instead of json?

Reply:
As I mentioned in my comment: my initial implementation was based on the `third_party/jsoncpp` library. The main issue with that library that when the Json object is parsed it is sorted in alphabetical order. For example the following Json object:
```
{
    1:INT: 42,
    2:UINT: 56,
    12:STRING: "test"
}
```

will be parsed as:
```
{
    1:INT: 42,
    12:STRING: "test",
    2:UINT: 56
}
```

This is not acceptable for our use case as the implementation MUST preserve order of elements. The new library that I am using does that properly. 

I see that other parts of SDK including some of the third_party libraries are using the `third_party/jsoncpp` library so I don't know if there is a way to get rig of one of them. As list I don't see an easy way to do that right now.